modules/restore-remove-failed-master.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-failed-master.adoc
+
+[id="restore-remove-failed-master_{context}"]
+= Removing a failed master host from the etcd cluster
+
+Follow these steps to remove a failed master host from the etcd cluster.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* SSH access to an active master host.
+
+.Procedure
+
+. Access an active master host.
+
+. Run the `etcd-member-remove.sh` script and pass in the name of the etcd member to remove.
++
+----
+$ sudo -E etcd-member-remove.sh etcd-member-ip-10-0-147-172.us-east-2.compute.internal
+Downloading etcdctl binary..
+etcdctl version: 3.3.10
+API version: 3.3
+etcd client certs already backed up and available ./assets/backup/
+Member 23e4736df4451b32 removed from cluster 6e25bab1bb556673
+etcd member etcd-member-ip-10-0-147-172.us-east-2.compute.internal with 23e4736df4451b32 successfully removed..
+----
++
+// TODO: Mention to use 'oc get pods -n openshift-etcd' to get name? Or not necessarily?
```

Comment:
@retroflexer In your draft, you first mentioned to check "oc get pods -n openshift-etcd" to see the pod list. But you said that the etcd member name wasn't necessarily the same as the etcd name, so I didn't include that. Is that okay to leave off then?